docs/server_guide.md

```diff
@@ -105,6 +105,7 @@ PrometheusPushGatewayMetricReporter is one of the built-in metrics reporter, whi
 
 |Property Name|Default| 	Description                                                                                                                                                                                                                                                                                                                          |
 |---|---|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|rss.metrics.reporter.class|org.apache.uniffle.common.metrics.<br/>prometheus.PrometheusPushGatewayMetricReporter|The class of metrics reporter.|
```

Comment:
Because the class name is too long, the length occupied by this column is too long, and this table needs to be scrolled, making it inconvenient to view.